plugins/filter_kubernetes/kube_meta.c

```diff
@@ -87,14 +89,104 @@ static int file_to_buffer(const char *path,
     return 0;
 }
 
+/* Run command to get Kubernetes authorization token */
+static int get_token_with_command(const char *command, 
+                                  char **out_buf, size_t *out_size)
+{
+    FILE *fp;
+    char buf[FLB_KUBE_TOKEN_BUF_SIZE];
+    char *temp;
+    char *res;
+    size_t size = 1;
```

Comment:
why does `size` start off at 1? I think this means in the first iteration of the while loop, you only realloc `res` to be +1 byte in size? That seems very inefficient. A bigger value would be better?